dataland-document-manager/src/main/kotlin/org/dataland/documentmanager/controller/DocumentController.kt

```diff
@@ -47,4 +59,51 @@ class DocumentController(
             .header(HttpHeaders.CONTENT_TYPE, "${document.type.mediaType}")
             .body(documentContent)
     }
+
+    override fun patchDocumentMetaInfo(
+        documentId: String,
+        documentMetaInfoPatch: DocumentMetaInfoPatch,
+    ): ResponseEntity<DocumentMetaInfoResponse> {
+        if (documentMetaInfoPatch.isNullOrEmpty()) {
```

Comment:
Bei meinem Test über die Swagger UI bin ich immer in diese Exception gelaufen. Egal wie das Patch-Objekt aussah.